examples/NonArduino/Tock/libtockHal.h

```diff
@@ -190,6 +190,10 @@ class TockHal : public RadioLibHal {
     void spiEnd() {
     }
 
+    void yield() {
+      yield();
+    }
+
```

Comment:
Will this call a global yield or will it recursively call itself? If the intention is to call a global function, then I would suggest explicitly specifying it with `::yield();`.